client/package.json

```diff
@@ -1,7 +1,9 @@
 {
   "name": "name-request",
-  "version": "0.3.3",
+  "version": "0.3.4",
   "private": true,
+  "appName": "Name Request UI",
+  "sbcName": "SBC Common Components",
```

Comment:
Cool, haven't seen this one before.

Reply:
I added this to our other projects a while back.